showkase-processor/src/main/java/com/airbnb/android/showkase/processor/ShowkaseProcessor.kt

```diff
@@ -131,13 +134,21 @@ class ShowkaseProcessor @JvmOverloads constructor(
     }
 
     private fun processPreviewAnnotation(roundEnvironment: XRoundEnv): Set<ShowkaseMetadata.Component> {
-        val skipPrivatePreviews = environment.options["skipPrivatePreviews"] == "true"
+        val skipPrivatePreviews = environment.options["skipPrivatePreviews"].toBoolean()
         val requireShowkaseComposableAnnotation =
-            environment.options["requireShowkaseComposableAnnotation"] == "true"
+            environment.options["requireShowkaseComposableAnnotation"].toBoolean()
 
         if (requireShowkaseComposableAnnotation) return emptySet()
 
         return roundEnvironment.getElementsAnnotatedWith(PREVIEW_CLASS_NAME)
+            .asSequence()
```

Comment:
For consistency, we should do this for all annotations (including ShowkaseComposable and custom annotations)

Reply:
Also maybe a comment explaining what it's trying to do as it's not obvious on first look for any future contributors.

Reply:
Re: adding it to everything else, it seems like other methods aren't affected, or at least not as reliant on ordering as this one.

I just extracted this function and started adding it to other call-sites in this file where annotations are queried, and it doesn't seem to be affecting anything (as in the tests still pass), so I wonder if it would just be an unnecessary processing step? Happy to go either way, but just wondering it's worth it